bugbot/rules/component.py

```diff
@@ -77,6 +82,16 @@ def get_bz_params(self, date):
         }
 
     def get_bugs(self, date="today", bug_ids=[]):
+        def select_threshold(bug_data):
+            if bug_data["class"] == "Fenix" or bug_data["class"] == "General":
+                return self.general_confidence_threshold
+            else:
+                return self.component_confidence_threshold
+
+        def meets_threshold(bug_data):
+            threshold = select_threshold(bug_data)
+            return bug_data["prob"][bug_data["index"]] >= threshold
```

Comment:
Do we really need the logic here to be in two functions?

Reply:
You're right, it's probably more clear to have it one function: 805066fecf101b890c0aa47115e8b339e8ccac1e